core/node/via_btc_sender/src/btc_inscription_aggregator.rs

```diff
@@ -128,13 +128,13 @@ impl ViaBtcInscriptionAggregator {
 
     // Todo: call indexer to fetch  the data
     async fn get_bootloader_code_hash(&self) -> anyhow::Result<H256> {
-        let hex_str = "0000000000000000000000000000000000000000000000000000000000000000";
+        let hex_str = "010008e742608b21bf7eb23c1a9d0602047e3618b464c9b59c0fba3b3d7ab66e";
```

Comment:
so it will remain constant, right?

Reply:
yes, for this phase i just keep it constant, but for next phases i will change this to use correct db table.